### ISSUE TITLE
Bug: Can't load Brenda Tissue Ontology

### ISSUE THREAD

**Issue by kalugny** [dhimmel/obonet#10]
```python console
>> bto = obonet.read_obo('http://data.bioontology.org/ontologies/BTO/submissions/33/download?apikey=8b5b7825-538d-40e0-9e9e-5ab9274a9aeb')

---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-8-b0207a3a5973> in <module>()
----> 1 bto = obonet.read_obo('http://data.bioontology.org/ontologies/BTO/submissions/33/download?apikey=8b5b7825-538d-40e0-9e9e-5ab9274a9aeb')

/usr/local/lib/python3.5/dist-packages/obonet/read.py in read_obo(path_or_file)
     25     obo_file.close()
     26     graph = networkx.MultiDiGraph(
---> 27         name=header['ontology'],
     28         typedefs=typedefs,
     29         instances=instances,

KeyError: 'ontology'
```

This is the same if I download the file and use it locally

**Comment by dhimmel** [comment]
Thanks @kalugny for the report.

Here are the header fields set by in the BRENDA OBO file you're reading:

```
format-version: 1.2
date: 03:05:2016 09:39
saved-by: Marion, contact@brenda-enzymes.org
auto-generated-by: OBO-Edit 2.3.1
synonymtypedef: GE "LANGUAGE GERMAN"
synonymtypedef: SCI "SCIENTIFIC NAME"
default-namespace: BrendaTissueOBO
remark: www.brenda-enzymes.org
```

Notice that the `ontology` field is missing. The following obonet code is written to require the field to be set:

https://github.com/dhimmel/obonet/blob/1441dd85cfb9b96cb88a32028a29e8bfe228695a/obonet/read.py#L26-L30

I guess it's a bad assumption that all OBO files will have this `ontology` field. The fix is easy: we can do `header.get('ontology')` which will return None if the key doesn't exist. Let me know if you think there's a better / different solution. Should get this fix out by the end of the day.

**Comment by dhimmel** [comment]
@kalugny this should be fixed by https://github.com/dhimmel/obonet/commit/17be627bbf1cb449dbb908a0021fb3f6c3f54f50 which is in the just released [v0.2.3](https://pypi.org/project/obonet/0.2.3/). Let me know if you have any remaining issues! Will close for now.
